models/user.go

```diff
@@ -2,16 +2,22 @@ package models
 
 import "strings"
 
+// UserId is used to symbolize that this int64 is a specifically a UserId
 type UserId int64
 
+// EmailAddress is a wrapper struct of the private string emailAddress to make
```

Comment:
seems useful to describe it's purpose, but describing what it is seems unnecessary